src/api/api.js

```diff
@@ -1,22 +1,35 @@
 import store from '../store/store';
 import axios from 'axios';
 
-const baseURL = 'http://198.199.100.209:5005/api';
-const getHeaders = () => {
+// const baseURL = 'http://198.199.100.209:5005/api';
+const baseURL = 'http://localhost:5005/api/v1.0';
+const getConfig = () => {
    return {
-      'headers': {
+      headers: {
          'Content-Type': 'application/json',
+         'Authorization': 'token 10cbb8be976d8db1efcca8c60fd7e3594f74772fd877d36afc825ed6df3b7829'
          // Need to implement login register
          // 'Authorization': store.getState().auth.token
-      }
+      },
+      data: {}
    }
 };
+
 const urls = {
-   charts: baseURL + '/charts'
+   charts: baseURL + '/charts',
+   transactions: baseURL + '/transactions'
 };
 
-const getStockHistory = async (ticker, length) => {
-   return await axios(urls.charts + '/${ticker}?length=${length}');
-}
+export const getStockHistory = async (ticker, length) => {
+   return await axios.get(urls.charts + `/${ticker}?length=${length}`, getConfig());
+};
 
-export { getStockHistory };
+export const transaction = async (ticker, shareCount, action, portfolioId) => {
+   console.log(shareCount);
```

Comment:
Still need this logging?

Reply:
Nope!

Reply:
I can get down with that. I will need to change the other api function to also do that so it is consistent. It's going to be important to comment the params, otherwise no one will know what params are needed.